bigcache.go

```diff
@@ -13,13 +14,13 @@ const (
 // It keeps entries on heap but omits GC for them. To achieve that operations on bytes arrays take place,
 // therefore entries (de)serialization in front of the cache will be needed in most use cases.
 type BigCache struct {
-	shards       []*cacheShard
-	lifeWindow   uint64
+	Shards       []*cacheShard
+	LifeWindow   uint64
 	clock        clock
 	hash         Hasher
-	config       Config
-	shardMask    uint64
-	maxShardSize uint32
+	Config       Config
+	ShardMask    uint64
+	MaxShardSize uint32
```

Comment:
Why do you changed visibility of this fields? This makes PR unreadable and cover main purpose of this PR.